api/helpers/Logger.js

```diff
@@ -1,9 +1,12 @@
 const winston = require('winston');
```

Comment:
@gkallenberg thanks for the idea of updating this file. Hopefully, we can create a general pattern or package each app can import rather than copying and pasting this file.